preload.js

```diff
@@ -32,7 +32,11 @@ document.addEventListener("DOMContentLoaded", () => {
 	// inject song-info provider
 	const songInfoProviderPath = path.join(__dirname, "providers", "song-info-front.js")
 	fileExists(songInfoProviderPath, require(songInfoProviderPath));
-	
+
+	// inject front logger
+	const loggerPath = path.join(__dirname, "providers", "front-logger.js")
+	fileExists(loggerPath, require(loggerPath));
```

Comment:
nit: contrary to plugins, we are sure this file exists so we can just require it directly!